app/src/util/helpers.ts

```diff
@@ -22,3 +22,27 @@ export function getCurrentVersion(): string {
   const version = process.env.APP_VERSION!;
   return version;
 }
+
+export function shortenNumber(number: number): string {
+  if (number >= 1e9) {
+    return (number / 1e9).toFixed(1); // Billion
+  } else if (number >= 1e6) {
+    return (number / 1e6).toFixed(1); // Million
+  } else if (number >= 1e3) {
+    return (number / 1e3).toFixed(1); // Thousand
+  } else {
+    return number.toString();
+  }
+}
+
+export function getShortenNumberUnit(number: number): string {
+  if (number >= 1e9) {
+    return "B"; // Billion
+  } else if (number >= 1e6) {
+    return "M"; // Million
+  } else if (number >= 1e3) {
+    return "K"; // Thousand
+  } else {
+    return number.toString();
```

Comment:
```suggestion
    return "";
```

I think in this case you will print the number twice otherwise.